### DISCUSSION TITLE
Member avatar url

### DISCUSSION THREAD

**Discussion by vitiacat** [Rapptz/discord.py#7233] [Q&A] [answered]
In alpha 2.0 'Member' object has no attribute 'avatar_url'

how to get member avatar url?

**Comment by lgaan** [comment, marked as answer]
The alpha comes with a lot of breaking changes, ensure you read the [2.0 documentation when migrating](https://discordpy.readthedocs.io/en/master).

With that said, `discord.Asset` now has a `url` attribute. (`Member.avatar.url`)

**Reply by vitiacat** [reply]
Thanks!
